server/backends/redis_key_val_store/redis_key_val_store.go

```diff
@@ -0,0 +1,40 @@
+package redis_key_val_store
+
+import (
+	"context"
+	"time"
+
+	"github.com/buildbuddy-io/buildbuddy/server/util/status"
+	"github.com/go-redis/redis"
+)
+
+const (
+	ttl = 3 * 24 * time.Hour
+)
+
+type store struct {
+	rdb *redis.Client
+}
+
+func New(rdb *redis.Client) *store {
+	return &store{rdb}
+}
+
+func (s *store) Set(ctx context.Context, key string, val []byte) error {
+	if val == nil {
+		s.rdb.Del(ctx, key)
```

Comment:
does this return an error that you can return?

Reply:
Yep, done